content_scripts/storageLoader.js

```diff
@@ -235,10 +236,10 @@ async function loadSave() {
      *  if it doesn't then it creates one)
      * and then it initialize the page according to the instructions given in the loaded settings.
      */
-    var moodleBoosterData = localStorage.getItem('MoodleBooster');   // Load MoodleBooster's data from the localStorage
+    var moodleBoosterData = localStorage.getItem('MoodleBooster' + currentYear);   // Load MoodleBooster's data from the localStorage
```

Comment:
Please add error handling for a situation that the currentYear not found.
I think of just putting the key 'MoodleBooster'